frontend/src/pages/GraphPF/styles/styleNode.tsx

```diff
@@ -105,6 +117,18 @@ const StyleNodeComponent: React.FC<StyleNodeProps> = ({ element, ...rest }) => {
     data.onHover(element, false);
   };
 
+  const nodeClass = kialiStyle({
+    $nest: {
+      '&.pf-m-hover': {
+        cursor: 'pointer'
+      },
+      '&.pf-m-selected .pf-topology__node__background': {
+        stroke: PFColors.Active,
```

Comment:
To get the alternative node selection with the status color, just remove this line.